packages/deploy/src/api/platform.ts

```diff
@@ -0,0 +1,15 @@
+import { BaseApiClient } from 'defender-base-client';
+
+export class PlatformApiClient extends BaseApiClient {
+  protected getPoolId(): string {
+    return process.env.DEFENDER_ADMIN_POOL_ID || 'us-west-2_94f3puJWv';
```

Comment:
`DEFENDER_ADMIN_POOL_ID`? Let's go with `PLATFORM_POOL_ID`, etc. Also bear in mind the API mapping issue between staging and prod.